src/OversetSimulation.cpp

```diff
@@ -149,9 +149,12 @@ void OversetSimulation::run_timesteps(const int add_pic_its, const int nsteps)
         for (auto& ss : m_solvers) ss->call_advance_timestep();
 
         if (add_pic_its > 0) {
-            exchange_solution();
-            for (auto& ss : m_solvers)
-                ss->call_additional_picard_iterations(add_pic_its);
+            for (int n = 0; n < add_pic_its; ++n) {
+                for (auto& ss : m_solvers) {
+                    exchange_solution();
+                    ss->call_additional_picard_iteration(n);
```

Comment:
Is n getting used anywhere? I would still keep `n` user defined until some guideline has been established on how many picards are sufficient to drop the nonlinear residual norm by 2 orders of magnitude on the nalu-wind side. At the very least, would keep this number at 2, not 1.